lra/certifications/index.html.haml

```diff
@@ -0,0 +1,14 @@
+---
+layout: rightcol
+title: Community
+---
+
+%h1
+  LRA Certifications
+
+
+%h2
+  %a(href="/lra/certifications/microprofile-lra-1.0/CCR-results") LRA 1.0 Compatible Certification Request (CCR)
```

Comment:
the link needs to be relative to the directory (ie no leading `/lra/certifications/`) and with an index.html, ie:

`%a(href="microprofile-lra-1.0/CCR-results/index.html">LRA 1.0 Compatible Certification Request (CCR)</a>`

Reply:
@xstefank your last push didn't add index.html to the url so that when you click on hyperlink it takes the user to a directory files list page not the actual index.html